reactive/coredns.py

```diff
@@ -14,9 +16,33 @@ def start_charm():
     image_info = layer.docker_resource.get_info('coredns-image')
 
     config = hookenv.config()
-    port = config.get('port')
 
+    corefile = Path('files/CoreFile').read_text() % config
+
+    print('COREFILE: {}'.format(corefile))
+    print('PORT: {}'.format(hookenv.config('port')))
```

Comment:
Should remove prints and either replace them with logs or just remove them if they were for debugging.